release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -293,6 +293,68 @@ This dashboard provides API server metrics, such as:
 
 // TODO: Link to troubleshooting using this dashboard section once it is available
 
+[id="ocp-4-7-monitoring-alerting-rule-changes"]
+==== Alerting rule changes
+
+{product-title} 4.7 includes the following alerting rule changes:
+
+* The `etcdBackendQuotaLowSpace` alert is added. The critical alert provides notification if the database size of an etcd cluster exceeds the defined quota on an etcd instance.
+* The `etcdHighFsyncDurations` alert is added. The critical alert provides notification if the 99th percentile fsync durations of an etcd cluster are too high.
+* The `etcdExcessiveDatabaseGrowth` alert is added. The warning alert provides notification if there is an observed surge in etcd writes that caused a 50% increase in database size on an etcd instance over a four-hour period.
+* The `ThanosSidecarPrometheusDown` alert is added. The critical alert provides notification that the Thanos sidecar cannot connect to Prometheus.
+* The `ThanosSidecarUnhealthy` alert is added. The critical alert provides notification that the Thanos sidecar is unhealthy for a specified amount of time.
+* The `NodeClockNotSynchronising` alert is updated to prevent false positives in environments that use the chrony time service (chronyd).
+* The `NodeNetworkReceiveErrs` alert is updated to ensure that the alert does not fire when only a small number of errors are reported. The rule now uses the ratio of errors to total packets instead of the absolute number of errors.
+* The `NodeNetworkTransmitErrs` alert is updated to ensure that the alert does not fire when only a small number of errors are reported. The rule now uses the ratio of errors to total packets instead of the absolute number of errors.
+
+[NOTE]
+====
+Red Hat does not guarantee backward compatibility for metrics, recording rules, or alerting rules.
+====
+
+[id="ocp-4-7-monitoring-stack-dependency-version-updates"]
+==== Version updates to monitoring stack components and dependencies
+
+{product-title} 4.7 includes version updates to the following monitoring stack components and dependencies:
+
+* The Prometheus Operator is now on version 0.44.1.
+* Thanos is now on version 0.17.2.
+* `kube-prometheus` is now on release 0.7.
```

Comment:
I don't think it is needed, we could remove it

Reply:
Thank you, @juzhao. @simonpasquier or @s-urbaniak, can you please confirm. This is in the CHANGELOG.

Reply:
I'd agree with @juzhao, kube-prometheus isn't an operand, it's only configuration.